README.md

```diff
@@ -74,7 +72,8 @@ $ make upgrade
 
 ## Contributing
 
-Read the [CONTRIBUTING](CONTRIBUTING) file and use GitHub pull request to
-contribute changes and improvements.
+Feel free to submit a pull request to contribute changes and improvements.
+
+## License
 
-For other inquiries, contact the zonedstorage.io team at zonedstorage@wdc.com.
+The site source code is released as open source software under the GPL v3 license, see the LICENSE file in the project root for the full license text.
```

Comment:
Can you make LICENSE a link to the file ?

Reply:
Yep

Reply:
I do not see "LICENSE" being a link... Make it [LICENSE](license.md) and that should do it.